internal/batches/service/service.go

```diff
@@ -397,7 +397,7 @@ on:
 steps:
```

Comment:
this must be reverted to spaces, YAML doesn't like tabs.

Reply:
😬 What auto-formatter is switching them back to tabs? Could we tell it to ignore this file in code somehow?

Reply:
Oh I missed that. I'll update now.

I guess I got confused because there was a mix of spaces and tabs so I just went with tabs for everything .

Reply:
Actually, i did that manually - there was an issue where i added a new field to the graphql query and that made use of tabs, while the rest of the query made use of spaces. I adjusted the whole file to use tabs and that's where the problem is, I'll just revert that commit so it doesn't break things.

Reply:
Reverted.